src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/PolicyVersionDifferencesTest.kt

```diff
@@ -61,6 +62,24 @@ class PolicyVersionDifferencesTest() {
     showDiffs(diffs)
   }
 
+  @Test
+  fun checkCodeVsJson() {
+    assertThat(licencePolicyService.policyByVersion("1.0"))
+      .usingRecursiveComparison()
+      .withStrictTypeChecking()
+      .isEqualTo(POLICY_V1_0)
```

Comment:
these tests prove the json and kotlin versions are equivalent and will ensure that the json and kotlin representation stay in sync

Reply:
(the recursive comparison bit just gives better feedback of the specific level where difference occur,  the strict typechecking is that it does instance checking too)